### PR TITLE
Enable resetting non-current branch without checking it out

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ This action always performs a `--ff-only` (fast-forward only) pull which makes h
 <br/>
 
 If a given branch is not in sync with its remote, you can reset it to point to the exact same commit as the remote.
-From a right-click context menu select `Checkout and Reset Branch to Remote`.<br/>
+From the right-click context menu, select `Reset Branch to Remote`.<br/>
```
Comment (PawelLipski) [inline README.md:168]: I'll regen the gif as well

File: uiTests/src/test/java/com/virtuslab/gitmachete/uitest/UITestSuite.java [hunk 2]
```diff
@@ -59,20 +59,7 @@ public void openProjectAndAwaitIdle() {
...
-
-    runJs("project.toggleListingCommits()");
-    int branchAndCommitRowsCount = callJs("project.refreshModelAndGetRowCount()");
-    // 6 branch rows + 7 commit rows
-    Assert.assertEquals(13, branchAndCommitRowsCount);
-  }
-
-  @Test
-  public void skipNonExistentBranches() {
+  public void skipNonExistentBranchesAndToggleListingCommits() {
```
Comment (PawelLipski) [inline uiTests/src/test/java/com/virtuslab/gitmachete/uitest/UITestSuite.java:62]: Merged the tests... to limit test running time at least a bit (since there's a significant overhead on creating/opening a new project in each test)

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java [hunk 3]
```diff
@@ -124,23 +126,34 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       return;
     }
 
+    var localBranch = getGitMacheteBranchByNameWithLogging(anActionEvent, branchName).getOrNull();
+    if (localBranch == null) {
+      VcsNotifier.getInstance(project).notifyError(VCS_NOTIFIER_TITLE,
+          "Internal error occurred. For more information see IDE log file");
```
Comment (micpiotrowski) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java:132]: Yeah, this kind of messages introduced by me was not the best from my part :disappointed: 
Let's reword this to something more useful like `Can't get a local branch '${branchName}'` or something similar.

File: README.md (hunk 1 above)
Comment (PawelLipski) [reply README.md:168]: Done

**Review by mkondratek — APPROVED** [review]
fine
